go.mod

```diff
@@ -1,25 +1,26 @@
 module github.com/k8snetworkplumbingwg/kubemacpool
 
-go 1.12
+go 1.13
```

Comment:
I think we should align with CI team using go 1.12.8 or at lease make sure we wont break anything,
if we dont then we can even use the latest version

Reply:
Let me try to re-generate with go 1.12 maybe go mod tidy find a compatible version for libraries.

Reply:
@ormergi I have being able to to goback to go 1.12 looks like we have to use kustomize v3 [1]

[1] https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md

I will re-organize this PR it's a mess already.